java/clients/src/main/java/sleeper/clients/status/report/FilesStatusReport.java

```diff
@@ -91,7 +91,7 @@ private static FileStatusReporter getReporter(String outputType) {
     }
 
     public void run() throws StateStoreException {
-        FileStatus fileStatus = fileStatusCollector.run(this.maxNumberOfFilesWithNoReferencesToCount);
+        TableFilesSummary fileStatus = fileStatusCollector.run(this.maxNumberOfFilesWithNoReferencesToCount);
```

Comment:
Rename variable to `tableFilesSummary`?

Reply:
I'm going to rename it back to FileStatus and do this in a separate issue:
- https://github.com/gchq/sleeper/issues/1716